### DISCUSSION TITLE
Problem with SophiApp

### DISCUSSION THREAD

**Discussion by serge5183** [Sophia-Community/SophiApp#209] [Show and tell]
I'm running Windows 11 24h2 and I can't use it. I'm up to date with the Windows update. I enclose an image of the message 

Sincerely 
![Capture d'écran 2024-10-27 103939](https://github.com/user-attachments/assets/9c0b05a1-2abd-4de0-af6a-21a183e599ce)

**Comment by farag2** [comment]
https://github.com/Sophia-Community/SophiApp?tab=readme-ov-file#usage
